functions/v2/pubsub/subscribe-to-topic/src/main/java/functions/SubscribeToTopic.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package functions;
+
+// [START functions_pubsub_subscribe]
+
+import com.google.events.cloud.functions.CloudEventsFunction;
```

Comment:
Does it make sense to use [this type](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/functions/pubsub/subscribe-to-topic/src/main/java/functions/SubscribeToTopic.java#L23) here?

Reply:
I used CloudEventsFunction here because it was being used in the audit logs example, I'm not sure which would be best here.